scripts/mkcloudcloud/heat-setup.sh

```diff
@@ -6,7 +6,17 @@ if [ -z "$namespace" -o -z "$openstacksshkey" ] ; then
 fi
 
 stackname=mkcc-$namespace
-openstack stack create --wait -t setup.yaml --parameter key_name=$openstacksshkey --parameter namespace=$namespace $stackname
+if openstack stack show $stackname >/dev/null; then
+  echo "Stack $stackname already exists, deleting..."
+  until openstack stack delete -y --wait $stackname; do
```

Comment:
Shouldn't this and the one further down stop trying after some loops?

Reply:
`--wait` will wait till the action is done or else fail.
we can let the script fail if either of the waits fail or just continue

Reply:
As I mentioned in the other comment, this was due to `stack delete` being executed before. You cannot create the stack straight away.